tests/testthat/test-dice.R

```diff
@@ -84,10 +81,15 @@ test_that("formula interface", {
   )
   
   expect_true(all(profileApply(s4, nrow) <= 31))
+})
   
+test_that("formula interface (with non-standard evaluation)", {
+  # NSE (these work interactively, but not in testthat env)
+  .slices <-  0:30
```

Comment:
Is this just remind us about NSE / testthat?

Reply:
Well, I suppose it is a reminder of the syntax in general, as it _can_ be used interactively. 
Mainly was just showing that the loss of ability to use NSE in this context is not a regression from slice->dice. 

We can comment it out / get rid of it so we don't have unnecessary number of tests of "expected" behavior

And when slice is deprecated or if removed from namespace would need modification anyway.

Reply:
cool, thanks